packages/kotti-ui/source/kotti-field-select/types.ts

```diff
@@ -12,10 +12,12 @@ export namespace KottiFieldMultiSelect {
 }
 
 export namespace Shared {
-	export type Entry = {
+	export type DefaultValue = string | number | boolean | symbol | null
```

Comment:
I think this doesn’t need to be exported, as `Kotti.Field*Select.Value` already is enough to refer to it if needed. Let’s keep it minimal and inline it instead